sway/config/input.c

```diff
@@ -148,6 +148,8 @@ struct input_config *store_input_config(struct input_config *ic) {
 		merge_wildcard_on_all(ic);
 	}
 
+	bool type_wildcard = strncmp(ic->identifier, "type:", 5) == 0;
```

Comment:
If this is a type config, it should be merged on all configs of the type similar to the wildcard. Note that this will not have any effect in the config since the inputs do not yet exist so it will have to be handled on retrieval as well